inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -311,9 +335,17 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
 
     writeBits(header, os);
 
+    for (auto &name : inputNames) {
+        writeBits(name.size(), os);
```

Comment:
low exposure but - sizeof(size_t) may be different than 8 no some configurations...